utils/wallet/wallet.go

```diff
@@ -6,19 +6,21 @@ import (
 	"time"
 
 	"github.com/brave-intl/bat-go/utils/altcurrency"
+	uuid "github.com/satori/go.uuid"
 	"github.com/shopspring/decimal"
 )
 
 // Info contains information about a wallet like associated identifiers, the denomination,
 // the last known balance and provider
 type Info struct {
-	ID            string                   `json:"paymentId" valid:"uuidv4,optional" db:"id"`
-	Provider      string                   `json:"provider" valid:"in(uphold)" db:"provider"`
-	ProviderID    string                   `json:"providerId" valid:"uuidv4" db:"provider_id"`
-	AltCurrency   *altcurrency.AltCurrency `json:"altcurrency" valid:"-"`
-	PublicKey     string                   `json:"publicKey,omitempty" valid:"hexadecimal,optional" db:"public_key"`
-	LastBalance   *Balance                 `json:"balances,omitempty" valid:"-"`
-	PayoutAddress *string                  `json:"anonymousAddress" valid:"uuidv4,optional" db:"payout_address"`
+	ID                string                   `json:"paymentId" valid:"uuidv4,optional" db:"id"`
+	Provider          string                   `json:"provider" valid:"in(uphold)" db:"provider"`
+	ProviderID        string                   `json:"providerId" valid:"uuidv4" db:"provider_id"`
+	AltCurrency       *altcurrency.AltCurrency `json:"altcurrency" valid:"-"`
+	PublicKey         string                   `json:"publicKey,omitempty" valid:"hexadecimal,optional" db:"public_key"`
+	LastBalance       *Balance                 `json:"balances,omitempty" valid:"-"`
+	ProviderLinkingID *uuid.UUID               `json:"providerLinkingId" valid:"-" db:"provider_linking_id"`
+	AnonymousAddress  *uuid.UUID               `json:"anonymousAddress" valid:"-" db:"anonymous_address"`
```

Comment:
I believe since they are of *uuid.UUID type they will either be nil (which is okay) or valid uuids.  The UUIDv4 govalidator would be good if these were string values.